ledger/sandbox/src/test/lib/scala/com/digitalasset/platform/sandbox/LedgerResource.scala

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
Why is this called `LedgerIT`? Shouldn't it be called `LedgerRespectsMRTIT` or similar?

Reply:
Because it's an integration test for the `Ledger` components `InMemoryLedger` and `SqlLedger`.

Reply:
Renamed it.